.github/ISSUE_TEMPLATE/merge-blocker.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Merge Blocker
+description: Merge blocker template for a release. Meant for use only by @openshift/openshift-team-windows-containers.
```

Comment:
Is there a way to enfore this: "Meant for use only by `@openshift/openshift-team-windows-containers`."

Reply:
I couldn't find anyway of enforcing that but I didn't dig deep into the docs. If you found something please let me know.